src/__tests__/setupTests.js

```diff
@@ -15,6 +15,15 @@ env.beforeEach(() => {
   // Fake timers let us flush Bridge operations between setup and assertions.
   jest.useFakeTimers();
 
+  const originalConsoleError = console.error;
+  console.error = (...args) => {
```

Comment:
Might need to `spyOn(console, 'error')` instead?
<img width="664" alt="Screen Shot 2019-04-18 at 9 23 56 AM" src="https://user-images.githubusercontent.com/29597/56375995-b54cd200-61bb-11e9-9b67-b3c718743468.png">

Reply:
Can I just flowfixme? I don't know how to use spyOn for this.

Reply:
Sure lol that's fine too